__tests__/fixtures/http_mocks.js

```diff
@@ -195,6 +195,14 @@ const mocks = {
       });
   },
 
+  head() {
+    return nock(mockHost)
+      .head('/')
+      .reply(200, null, {
```

Comment:
Why is second argument different from options reply ?

Reply:
Oh I didn't care it. I will fix